lib/app_files/app_credentials/welcome_note3_category1.dart

```diff
@@ -54,7 +54,7 @@ class Category1WelcomeNote3 extends StatelessWidget {
             SizedBox(height: Dimentions.height30),
             GestureDetector(
               onTap: () {
-                Get.toNamed(FormIntegrator.getDashboard());
+                Get.toNamed(FormIntegrator.getDashboard(0));
```

Comment:
actually i made this because to specially focus the main home page in the dashboard.